RxExample/RxExample/Examples/GitHubSearchRepositories/GitHubSearchRepositoriesViewController.swift

```diff
@@ -43,8 +43,9 @@ class GitHubSearchRepositoriesViewController: ViewController, UITableViewDelegat
 
 
```

Comment:
👍

At the beginning I wanted to put the tableView inside the capture list to the closure. However, the tableView is forced unwrapped optional and when you put it in the capture list it become the optional.

That's also a reason why I set type explicit in `let tableView: UITableView = self.tableView`